src/layouts/Layout.astro

```
@@ -2,7 +2,7 @@
 import { SITE } from "@config";
 import "@styles/base.css";
 import "@fontsource-variable/noto-sans-jp";
-// import { ViewTransitions } from "astro:transitions";
+import { ClientRouter } from "astro:transitions";
```

ESLint

[error] `astro:transitions` import should occur before import of `@config`

(import-x/order)